src/librustdoc/html/highlight.rs

```diff
@@ -29,6 +30,8 @@ crate struct ContextInfo<'a, 'b, 'c> {
     crate root_path: &'c str,
 }
 
+crate type DecorationInfo = FxHashMap<&'static str, Vec<(u32, u32)>>;
```

Comment:
I wrapped this with a struct. Since it's only one field, I didn't give it a name, but I can if you think it's clearer. I added documentation explaining the parts of the type.